### ISSUE TITLE
how to handle keyboard events with ui?

### ISSUE THREAD

**Issue by sergeytol** [andlabs/ui#382]
Is it possible to handle onKeyDown() and other keyboard events with ui somehow? Are there any examples of code?

**Comment by andlabs** [comment]
Yes, it's done via uiArea. I don't have any samples on hand, sorry.

**Comment by sergeytol** [comment]
for uiArea yes, but for other objects e.g uiMultilineEntry?

**Comment by andlabs** [comment]
Not yet, but you can detect when a change has happened to those.

**Comment by sergeytol** [comment]
got it, thanks so much for a quick answer! lovely UI, I'll hope the keyboard events will be implemented sooner or later
